iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -55,11 +55,13 @@ UnixDomainSocket::UnixDomainSocket(UnixDomainSocket&& other) noexcept
     *this = std::move(other);
 }
 
```

Comment:
Update copyright date. Unless we do not for comments only but I think those suppressions are non-trivial comments so it should be done if we want these copyrights.